hasjob/views/helper.py

```diff
@@ -431,7 +431,7 @@ def make_pay_graph(currency, posts, rmin=None, rmax=None, minposts=5):
 
 
 @job('hasjob')
-def save_jobview(event_session_id, jobpost_id, bgroup):
+def save_jobview(datetime, event_session_id, jobpost_id, bgroup):
```

Comment:
Don't use the parameter name `datetime`. There is already a symbol named `datetime` at module scope.